### DISCUSSION TITLE
上游的geoip.dat文件体积增大了6倍,有啥替代的geoip.dat文件么

### DISCUSSION THREAD

**Discussion by yeezylife** [IrineSistiana/mosdns#146] [Q&A] [answered]
https://github.com/Loyalsoldier/v2ray-rules-dat/releases/latest/download/geoip.dat
用的推荐的这个地址的geoip.dat文件..前两天突然体积增加到接近50M...

请问有替代方案么(体积小一点,效果差不多的geoip)

**Comment by y1zhou** [comment, marked as answer]
#206 里面有人推荐了这个：https://github.com/Loyalsoldier/geoip
